MachObfuscator/Options/Options.swift

```diff
@@ -140,6 +121,7 @@ extension Options {
           -q, --quiet             quiet mode, no output to stdout
           -v, --verbose           verbose mode, output verbose info to stdout
           -d, --debug             debug mode, output more verbose info to stdout
+          --dry-run               analyze only, do not save obfuscated images
```

Comment:
What do you think about changing `images` -> `files`? `MachObfuscator` also obfuscates NIBs.

Reply:
I agree. Changed.